integration-cli/docker_cli_build_test.go

```diff
@@ -863,10 +862,10 @@ func (s *DockerSuite) TestBuildCopyWildcard(c *check.C) {
 	ctx, err := fakeContext(fmt.Sprintf(`FROM busybox
 	COPY file*.txt /tmp/
 	RUN ls /tmp/file1.txt /tmp/file2.txt
-	RUN mkdir /tmp1
+	RUN sh -c "mkdir /tmp1"
```

Comment:
@duglin In response to your question about busybox and using `\`. It works as a path separator, but is explicitly called out as not valid, so the answer is no, but it works anyway.